integration/common/openlineage/common/provider/dbt/structured_logs.py

```diff
@@ -253,7 +260,7 @@ def parse_node_finished_event(self, event) -> RunEvent:
 
         event_type = RunState.COMPLETE
 
-        if node_status != "success":
+        if node_status not in ("success", "pass"):
```

Comment:
Not necessarily a thing to do now, but might be worth to have those statuses as an enum